openwisp_radius/templates/radius_accounting_start.html

```diff
@@ -1,5 +1,7 @@
 {% load i18n %}{% load l10n %}
+{% load humanize %}
 <div class="msg">
 <p>{% trans "A new session has been started for your account:" %} {{ user.get_username }}.</p>
 <p>{% trans "You can review your session to find out how much time and/or traffic has been used or you can terminate the session at any time by clicking on the button below." %}</p>
+{% if sesame_max_age %}<p>{% blocktrans with sesame_max_age|naturaltime as sesame_max_age %}NOTE: This link is valid only for {{sesame_max_age}}{% endblocktrans %}.</p>{% endif %}
```

Comment:
@Aryamanz29 did you tested email translation manually? I tested it in my local environment but it didn't work. Can you double check this? 

```html
<div class="msg">
<p>È iniziata una nuova sessione per il tuo account: user_pandafy.</p>
<p>Puoi monitorare il consumo di tempo e traffico della tua sessione o terminare la sessione in qualsiasi momento cliccando sul collegamento seguente.</p>
<p>NOTE: This link is valid only for tra 59 minuti.</p>
</div>
</div>
```

Reply:
> @Aryamanz29 did you tested email translation manually? I tested it in my local environment but it didn't work. Can you double check this?
> 
> ```
> <div class="msg">
> <p>È iniziata una nuova sessione per il tuo account: user_pandafy.</p>
> <p>Puoi monitorare il consumo di tempo e traffico della tua sessione o terminare la sessione in qualsiasi momento cliccando sul collegamento seguente.</p>
> <p>NOTE: This link is valid only for tra 59 minuti.</p>
> </div>
> </div>
> ```

@pandafy 
I've used the below setting while testing 🤔
```python
    with override_settings(SESAME_MAX_AGE=2*60*60, LANGUAGE_CODE='it'):
        with self.subTest(
                'it should check expiration text is present when SESAME_MAX_AGE is set'
            ):
            # Test code
```
It is showing translated text in my console :point_down: 
```html
<div class="msg">
<p>È iniziata una nuova sessione per il tuo account: tester.</p> 
<p>Puoi monitorare il consumo di tempo e traffico della tua sessione o terminare la sessione in qualsiasi momento cliccando sul collegamento seguente.</p> 
<p>NOTA: Questo link è valido solo per tra un&#x27;ora.</p> </div> </div> <div class="call-to-action"> <a href="https://wifi.openwisp.org/default/login/?sesame=e9Gy-IW1SF6EmS8wvv6wHlwHeqwgNu1f78M" class="btn">Gestisci sessione</a> 
</div>
```

Reply:
@Aryamanz29 the translation should work regardless of `LANGUAGE_CODE` setting. We translate the email context according to user preference (according to `accept-language` header of the user's requests.

Reply:
Tested with `wifi-login-pages` :point_down: 
![image](https://user-images.githubusercontent.com/56113566/156303419-227c6d4a-7f4d-4b69-b548-78b3f7814ca7.png)